src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/ExceptionProcessor.java

```diff
@@ -33,8 +38,8 @@ public void process(Exchange exchange) throws Exception {
             log.error("::::exception in route for data processing::::" + exception);
             exchange.getContext().getGlobalOptions().put(SCHEDULER_STATUS, FAILURE);
             exchange.getContext().getGlobalOptions().put(IS_EXCEPTION_HANDLED, TRUE.toString());
-            //check mail flag and send mail
-            emailService.sendEmail(exception.getMessage());
+            exchange.getContext().getGlobalOptions().put(ERROR_MESSAGE, exception.getMessage());
```

Comment:
exchange.getContext().getGlobalOptions() can be stored in one variable since it is used 3 times

Reply:
done